src/layouts/header.html

```diff
@@ -1,3 +1,48 @@
-<header>
-    dafjsdf
-</header>
\ No newline at end of file
+<header class="header-main">
+    <div class="container">
+      <!-- Start of header-top -->
+      <div class="header-main__top">
+        <a class="header-main__logo">
+            <img src="../assets/icons/ic-television.svg" alt="television" class="logo">
```

Comment:
You should wrap them with the `figure`

Reply:
I'll repair it before my report at tomorrow. Thank you !